c/c++.cpp

```diff
@@ -1,9 +1,9 @@
-#include <iostream> // include API
+#include <iostream> // Include the I/O library
 
 using namespace std;
 
-int main() // the main code portion of a C++ program
+int main(void) // The main function of a C++ program
 {
-   cout << "Hello World" << endl;  //print Hello World on the screen 
-   return 0; // conventional
+   cout << "Hello World" << endl;  // Print Hello World on the screen 
+   return 0;                       // Return 0 from the program (success)
```

Comment:
`return 0;` is optional/unnecessary from the `main()` function in C++. https://en.cppreference.com/w/cpp/language/main_function